### PR TITLE
feat: Native Support for Netflix OSS transformation

### PR DESCRIPTION
**Pull request by kmehant**
Signed-off-by: K mehant <411843@student.nitandhra.ac.in>

### CODE REVIEW

File: assets/filepermissions.yaml [hunk 1]
```diff
@@ -17,6 +17,7 @@
 "inbuilt/clusters/kubernetes.yaml" : 0644
 "inbuilt/clusters/openshift.yaml" : 0644
 "inbuilt/configs/dc.yaml" : 0644
+"inbuilt/transformers/.DS_Store" : 0644
```
Comment (ashokponkumar) [inline assets/filepermissions.yaml:20]: Can we remove this?

File: assets/filepermissions.yaml [hunk 2]
```diff
@@ -26,6 +27,7 @@
 "inbuilt/transformers/compose/composeanalyser/composeanalyser.yaml" : 0644
 "inbuilt/transformers/compose/composegenerator/composegenerator.yaml" : 0644
 "inbuilt/transformers/dockerfiledetector/dockerfiledetector.yaml" : 0644
+"inbuilt/transformers/executable/.DS_Store" : 0644
```
Comment (ashokponkumar) [inline assets/filepermissions.yaml:30]: Remove this

File: assets/filepermissions.yaml [hunk 3]
```diff
@@ -66,6 +68,7 @@
 "inbuilt/transformers/executable/dockerfile/rust/m2kdetect.sh" : 0755
 "inbuilt/transformers/executable/dockerfile/rust/rust.yaml" : 0644
 "inbuilt/transformers/executable/dockerfile/rust/templates/Dockerfile" : 0644
+"inbuilt/transformers/executable/s2i/.DS_Store" : 0644
```
Comment (ashokponkumar) [inline assets/filepermissions.yaml:71]: Remove this

File: go.mod [hunk 4]
```diff
@@ -33,6 +32,7 @@ require (
 	github.com/spf13/viper v1.8.0
 	github.com/tektoncd/pipeline v0.25.0
 	github.com/tektoncd/triggers v0.14.2
+	github.com/vifraa/gopom v0.1.0 // indirect
```
Comment (ashokponkumar) [inline go.mod:35]: I assume we are not using this rather your own version. So we can remove this reference.

File: internal/transformer/classes/analysers/zuulanalyser.go [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package analysers
+
+import (
+	//  "encoding/xml"
```
Comment (ashokponkumar) [inline internal/transformer/classes/analysers/zuulanalyser.go:20]: Remove all the commented lines

File: internal/transformer/classes/analysers/zuulanalyser.go [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+			namedServices[servicename] = append(namedServices[servicename], ct)
+		 }
+	 }
+	 return namedServices, nil, nil
+}
+
+// DirectoryDetect runs detect in each sub directory
+func (t *ZuulAnalyser) DirectoryDetect(dir string) (namedServices map[string]plantypes.Service, unnamedServices []plantypes.Transformer, err error) {
+
+
```
Comment (ashokponkumar) [inline internal/transformer/classes/analysers/zuulanalyser.go:119]: Remove the empty lines

**Review by ashokponkumar — CHANGES_REQUESTED** [review]

**Comment by kmehant** [comment]
@ashokponkumar Thank you for the review! I have addressed the changes!

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**